node/pkg/watchers/evm/watcher.go

```diff
@@ -288,6 +288,12 @@ func (w *Watcher) Run(parentCtx context.Context) error {
 			case <-ctx.Done():
 				return nil
 			case r := <-w.obsvReqC:
+				if r.ChainId > math.MaxUint16 {
+					logger.Error("chain id for observation request is not a valid uint16",
```

Comment:
does this need a `continue` or something afterwards? I think this just logs an error and then proceeds to use the value anyway :thinking: